src/rust/engine/fs/store/src/snapshot_ops.rs

```diff
@@ -228,7 +228,7 @@ pub trait SnapshotOps: Clone + Send + Sync + 'static {
   ) -> Result<DirectoryDigest, Self::Error> {
     let input_tree = self.load_digest_trie(directory_digest.clone()).await?;
     let path_stats = input_tree
-      .expand_globs(params.globs, SymlinkBehavior::Oblivious, None)
+      .expand_globs(params.globs, SymlinkBehavior::Aware, None)
```

Comment:
Hm, just confirming because I missed this in #18963. There's a `SymlinkBehaviour::Oblivious` below to compute `files` that's fed into the next call using `path_stats`. Is that correct?

Reply:
Yes. That one doesn't really make a difference. It's collecting the file digests from the subset. So being aware of not doesn't change the file digests collected 🤷‍♂️

Reply:
And to prove it, you can run the tests with either value and it passes